watchtower/comments_.py

```diff
@@ -48,7 +53,8 @@ def update_comments(user, project, auth=None, state="all", since=None,
     url = 'https://api.github.com/repos/{}/{}/issues/comments'.format(
         user, project)
     raw = _github_api.get_frames(auth, url, state=state, since=since,
-                                 max_page=max_page, per_page=per_page,
+                                 max_pages=max_pages, per_page=per_page,
```

Comment:
ah I bet we could have more reliability in our testing by using `since` along with `max_pages==1` yeah? Then we always know we'd get the same page.